src/System.Diagnostics.DiagnosticSource/src/System/Diagnostics/Activity.cs

```diff
@@ -392,13 +404,14 @@ private string GenerateRootId()
                 Interlocked.CompareExchange(ref s_uniqPrefix, GenerateInstancePrefix(), null);
             }
 #if DEBUG
-            string ret = s_uniqPrefix + "-" + OperationName + "-" + Interlocked.Increment(ref s_currentRootId).ToString("x") + '.';
+            string ret = s_uniqPrefix + "-" + OperationName.Replace('.', '-') + "-" + Interlocked.Increment(ref s_currentRootId).ToString("x") + '.';
 #else       // To keep things short, we drop the operation name 
             string ret = s_uniqPrefix + "-" + Interlocked.Increment(ref s_currentRootId).ToString("x") + '.';
 #endif
             return ret;
         }
 
+        [SecuritySafeCritical]
```

Comment:
That's happening because this project was previously copying the netstandard1.3 build to the net46 folder.  Now that you are building explicitly for netstandard1.3 you don't need to do that. See https://github.com/ericstj/corefx/commit/85e6fabc7eab650b80efa9902e17f56e17fb7654